README.md

```diff
@@ -31,66 +31,89 @@ The Perl container is set up so that Apache will load additionally provided .con
 
 ###Manual Installation: 
 1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+  -*NOTE*: OpenShift Online currently is using V2.
```

Comment:
just remove this entirely.  

add an intro that just says "You must have an openshift deployment that you have access to in order to deploy this app"